setup.py

```diff
@@ -194,7 +194,7 @@ def write_version(filename=os.path.join(*[my_dir, "airflow", "git_version"])):
     'azure-cosmos>=3.0.1'
 ]
 azure_data_lake = [
-    'azure-datalake-store>=0.0.45'
+    'azure-datalake-store>=0.0.45',
```

Comment:
👍 can we take this out altogether?

Reply:
In Airflow 2.X, the 3rd party provider dependencies will be removed into separate optional packages, but for now I just wanted to keep the diff smaller by fixing the constraint.

If we remove too many of the unnecessary dependencies, it might change a number of min/max constraints for other packages that I don't want to go through the work of validating version changes for when we're trying to get off of 1.10 altogether 😓 

But yes, it is kind of annoying that we need the azure dependencies.